src/ai/backend/manager/idle.py

```diff
@@ -231,14 +294,32 @@ async def update_app_streaming_status(
     ) -> None:
         pass
 
+    @classmethod
+    def get_report_key(cls, session_id: SessionId) -> str:
+        return f"session.{session_id}.{cls.name}.report"
+
     @abstractmethod
-    async def check_session(self, session: Row, dbconn: SAConnection, policy: Row) -> bool:
+    async def report_idle_check(
+        self, kernel: Row, dbconn: SAConnection, policy: Row, redis_obj: RedisConnectionInfo
+    ) -> bool:
```

Comment:
Main characteristic of this function is to determine if the kernel violates current idle checker policy. Please update the interface name so that users can understand express the characteristic at first glance.